src/bundle/Resources/public/scss/_warning-icon.scss

```diff
@@ -13,7 +13,7 @@
             justify-content: center;
             display: none;
 
-            .ez-icon {
+            .ez-icon.ez-icon--warning {
```

Comment:
Are you sure you haven't messed up some other place with this change?

Reply:
> Are you sure you haven't messed up some other place with this change?

there was a more general selector before

Reply:
It will lose styling for elements that don't have class `ez-icon--warning`.
Why do you need more specific selector?